enzyme/tools/enzyme-tblgen/enzyme-tblgen.cpp

```diff
@@ -2632,6 +2637,11 @@ void emitDiffUse(const RecordKeeper &recordKeeper, raw_ostream &os,
       break;
     }
     case IntrDerivatives: {
+      auto minVer = pattern->getValueAsInt("minVer");
```

Comment:
See above, there is prior support for passing min/max versions

Reply:
done